reference/components/document-manager/debugging.rst

```diff
@@ -0,0 +1,109 @@
+    | query.create         |
+    | query.create_builder |
+    | query.execute        |
+    | configure_options    |
+    +----------------------+
+
+Logging
+-------
+
+The Document Manager provides detailed logging about which subscribers are
+executed, the state of the event and the time taken by each event to be
+executed, for example:
+
+
+.. code-block:: bash
+
+    0.012195 S\C\C\D\S\ExtensionSubscriber           handlePersist        n:/cmf/sulu_io/contents/test1 d:0000000021b01e32000000005bcf8fba l:en p:/cmf/sulu_io/contents
+    0.000000 S\C\C\D\S\WebspaceSubscriber            handlePersist        n:/cmf/sulu_io/contents/test1 d:0000000021b01e32000000005bcf8fba l:en p:/cmf/sulu_io/contents
+    0.076923 S\C\C\D\S\ContentSubscriber             handlePersist        n:/cmf/sulu_io/contents/test1 d:0000000021b01e32000000005bcf8fba l:en p:/cmf/sulu_io/contents
+    0.000000 S\C\C\D\S\NavigationContextSubscriber   handlePersist        n:/cmf/sulu_io/contents/test1 d:0000000021b01e32000000005bcf8fba l:en p:/cmf/sulu_io/contents
+    0.000000 S\C\C\D\S\RedirectTypeSubscriber        handlePersist        n:/cmf/sulu_io/contents/test1 d:0000000021b01e32000000005bcf8fba l:en p:/cmf/sulu_io/contents
+    0.000000 S\C\C\D\S\WorkflowStageSubscriber       handlePersist        n:/cmf/sulu_io/contents/test1 d:0000000021b01e32000000005bcf8fba l:en p:/cmf/sulu_io/contents
+    0.000000 S\C\C\D\S\OrderSubscriber               handlePersist        n:/cmf/sulu_io/contents/test1 d:0000000021b01e32000000005bcf8fba l:en p:/cmf/sulu_io/contents
+    0.000000 S\C\C\D\S\RouteSubscriber               handlePersist        n:/cmf/sulu_io/contents/test1 d:0000000021b01e32000000005bcf8fba l:en p:/cmf/sulu_io/contents
+    0.000000 S\C\D\S\B\A\BlameSubscriber             handlePersist        n:/cmf/sulu_io/contents/test1 d:0000000021b01e32000000005bcf8fba l:en p:/cmf/sulu_io/contents
+    0.000000 S\C\D\S\B\A\TimestampSubscriber         handlePersist        n:/cmf/sulu_io/contents/test1 d:0000000021b01e32000000005bcf8fba l:en p:/cmf/sulu_io/contents
+    0.000000 S\C\D\S\B\M\NodeNameSubscriber          handleNodeName       n:/cmf/sulu_io/contents/test1 d:0000000021b01e32000000005bcf8fba l:en p:/cmf/sulu_io/contents
+    0.000000 S\C\D\S\B\M\UuidSubscriber              handleUuid           n:/cmf/sulu_io/contents/test1 d:0000000021b01e32000000005bcf8fba l:en p:/cmf/sulu_io/contents
+    0.000000 S\C\D\S\B\M\ParentSubscriber            handleChangeParent   n:/cmf/sulu_io/contents/test1 d:0000000021b01e32000000005bcf8fba l:en p:/cmf/sulu_io/contents
+
+Have a closer look:
+
+.. code-block:: bash
+
+    [1]      [2]                            [3]             [4]
+    0.012195 S\C\C\D\S\ExtensionSubscriber  handlePersist   n:/cmf/sulu_io/contents/test1 d:0000000021b01e32000000005bcf8fba l:en
+
+1. The time taken by the subscriber, expressed as a fraction of a second.
+2. The class name. The namespace is compressed to allow for greater
+   readability.
```

Comment:
Actually this is not true anymore. Thomas removed this, the full namespace is shown instead.